virtcontainers/factory/grpcCache/grpcCache.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2019 HyperHQ Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// grpcCache implements base vm factory that get base vm from grpc
+
+package grpcCache
```

Comment:
Please just call it `grpccache` or `grpc-cache`. No camel needed for package names. And I suppose it can be built on top of `cache` factory. Might be a task for future PRs.

Reply:
done.